src/app/campaigns/components/farm-welcome/farm-welcome.component.html

```diff
@@ -183,15 +177,15 @@
 </div>
 
 <div class="welcome_btn_launch" id="btn_launch_stat">
-  <button class="btnStart" routerLink="/ad-pools">Launch App</button>
+  <button class="btnStart" routerLink="/ad-pools">{{ 'Launch_App_Welcome' | translate }}</button>
 </div>
 
 <div class="mt-5 d-flex blog-bloc">
   <div class="col-4 d-none d-sm-block" style="left: 4%">
     <div class="row adpool-title mb-3">Blog</div>
     <div class="row blog-description mb-3">
-      To stay updated on SaTT's progress and be informed of trends in the
-      blockchain world, regularly visit our blog for updates.
+      {{ 'Blog_welcome' | translate}}
+      {{ 'Blog_welcome2' | translate}}
```

Comment:
In order to facilitate localization and internationalization, could we please place this entire block of text into a single i18n translation variable? This will allow us to translate the entire block of text at once, rather than dealing with individual sentences, ensuring a more consistent translation. Thank you.